docs/changes/coordinates/11444.feature.rst

```diff
@@ -0,0 +1,7 @@
+All representations now have a ``transform`` method, which allows them to be
+transformed by a 3x3 matrix in a Cartesian basis. By default, transformations
+are routed through ``CartesianRepresentation``. ``SphericalRepresentation`` and
+``PhysicssphericalRepresentation`` override this for speed and to prevent NaN
+leakage from the distance to the angular components.
+Also, add a utility function in ``matrix_utities`` to check whether a matrix is
```

Comment:
`matrix_utilities` is half-hidden, which is a bit strange. I think it is not bad to give it some more exposure, though if we do, might as well give the name of the routine.

Reply:
✅

Reply:
Does it not appear in the docs?